ci_tools/list_docs_tovalidate.py

```diff
@@ -6,55 +6,86 @@
 from pathlib import PurePath
 from git_evaluation_tools import get_diff_as_json
 
-parser = argparse.ArgumentParser(description='List the objects with docstrings in the files provided')
-parser.add_argument('gitdiff', metavar='gitdiff', type=str,
+def should_ignore(name):
+    '''
+        Determine if an object should be ignored from numpydoc validation.
+
+        Parameters
+        ----------
+        name : str
+            Absolute name of the inspected object.
+
+        Returns
+        -------
+        bool
+            True if object should be ignored, False otherwise.
```

Comment:
Talking about numpydoc... isn't there an excessive indent in this docstring? :sweat_smile:

Reply:
numpydoc doesn't complain about that. But it does complain about a lack of an extended summary